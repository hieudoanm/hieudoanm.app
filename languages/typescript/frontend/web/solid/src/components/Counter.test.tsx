import { render } from '@solidjs/testing-library';
import { describe, it, expect } from 'vitest';
import Counter from './Counter';

describe('Counter', () => {
  it('should assert some dummy assertion', () => {
    const { container } = render(() => <Counter />);
    expect(container).toMatchSnapshot();
  });
});
