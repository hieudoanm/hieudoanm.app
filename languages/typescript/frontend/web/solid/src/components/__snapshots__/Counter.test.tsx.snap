// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Counter > should assert some dummy assertion 1`] = `
<div>
  <button
    class="increment"
  >
    Clicks: 
    0
  </button>
</div>
`;
